db/migrate/20220509104200_create_goods_nomenclature_export_function.rb

```diff
@@ -0,0 +1,114 @@
+Sequel.migration do
+  up do
+    Sequel::Model.db.run('CREATE SCHEMA IF NOT EXISTS "public"')
+    Sequel::Model.db.run(function_body)
+  end
+
+  down do
+    Sequel::Model.db.run('DROP FUNCTION IF EXISTS "public.goods_nomenclature_export_new"')
+  end
+end
+
+private
+
+def function_body
```

Comment:
This is in the wrong namespace - this is at the global level, not within the database migration itself

I'd suggest just inlining this, ie 

```ruby
Sequel::Model.db.run <<~SQL.freeze
```